.github/workflows/ci.yml

```diff
@@ -25,10 +25,11 @@ jobs:
       # This should be php 7.3 & 8.0 in 4.10, and php 7.4 & 8.1 in 4.11+
       # Run recipe-cms testsuite because that's the most likely to have weird conflicts e.g. graphql
       extra_jobs: |
-        - php: 7.4
-          composer_args: --prefer-lowest
-          phpunit: true
-          phpunit_suite: recipe-cms
         - php: 8.1
           phpunit: true
           phpunit_suite: recipe-cms
+      # TODO: renable prefer-lowest build once CMS 5 beta is out
+      # - php: 8.1
+      #   composer_args: --prefer-lowest
+      #   phpunit: true
+      #   phpunit_suite: recipe-cms
```

Comment:
I've added this to https://github.com/silverstripe/gha-generate-matrix/issues/41